interface/batchcom/batchcom.php

```diff
@@ -24,280 +29,217 @@
-                ?>
-                </SELECT></td>
-           </tr><tr><td>
-        <!-- later gator
-        <br>Insurance: <SELECT multiple NAME="insurance" Rows="10" cols="20">
-
-                        </SELECT>
-        -->
-      <?php xl('And','e')?>:<INPUT TYPE="radio" NAME="and_or_app_within" value="AND" checked>, <?php xl('Or','e')?>:<INPUT TYPE="radio" NAME="and_or_app_within" value="OR"></td><td> <?php xl('Appointment within','e')?>:</td><td><INPUT TYPE='text' size='12' class='datepicker' NAME='app_s'></td><td>
-
-        <?php xl('And','e')?> :  <INPUT TYPE='text' size='12' class='datepicker' NAME='app_e'></td>
-     </tr><tr><td>
-
-     <?php xl('And','e')?>:<INPUT TYPE="radio" NAME="and_or_seen_since" value="AND" checked>, <?php xl('Or','e')?>:<INPUT TYPE="radio" NAME="and_or_seen_since" value="OR"></td><td> <?php xl('Seen since','e')?> :</td><td><INPUT TYPE='text' size='12' class='datepicker' NAME='seen_since'></td>
-  <td>&nbsp;</td>
-   </tr><tr><td>
-
-        <?php xl('And','e')?>:<INPUT TYPE="radio" NAME="and_or_not_seen_since" value="AND" checked>, <?php xl('Or','e')?>:<INPUT TYPE="radio" NAME="and_or_not_seen_since" value="OR"></td><td> <?php xl('Not seen since','e')?> :</td><td><INPUT TYPE='text' size='12' class='datepicker' NAME='not_seen_since'></td>
- <td>&nbsp;</td>
-   </tr><tr><td>
-        <?php xl('Sort by','e')?> :</td><td><SELECT NAME="sort_by">
-                <?php
-                foreach ($sort_by as $key => $value) {
-                    echo ("<option value=\"".$value."\">$key</option>");
-                }
-                ?>
-                </SELECT></td>
-     <td>&nbsp;</td><td>&nbsp;</td>
-       </tr><tr><td colspan='3'>
-    (<?php xl('Fill here only if sending email notification to patients','e')?>)</td>
-   <td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
-   </tr><tr><td>
-    <?php xl('Email Sender','e')?> :</td><td><INPUT TYPE="text" NAME="email_sender" value="your@example.com"></td>
-  <td>&nbsp;</td><td>&nbsp;</td>
-   </tr><tr><td>
-    <?php xl('Email Subject','e')?>:</td><td><INPUT TYPE="text" NAME="email_subject" value="From your clinic"></td>
-  <td>&nbsp;</td><td>&nbsp;</td>
-  </tr><tr><td colspan='3'>
-    <?php echo xlt('Email Text, Usable Tag: ***NAME*** , i.e. Dear ***NAME***{{Do Not translate the ***NAME*** elements of this constant.}}')?></td>
-   <td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr>
-   <tr><td colspan='4'>
-    <TEXTAREA NAME="email_body" ROWS="8" COLS="40"></TEXTAREA></td>
-    <td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
-     </tr><tr><td>
-
-    <INPUT TYPE="submit" name="form_action" value=<?php xl('Process','e','\'','\''); ?>> </td><td><?php xl('Process takes some time','e')?></td> <td>&nbsp;</td><td>&nbsp;</td></tr>
-</table>
-</div>
-</div>
-</FORM>
-
+        process.addEventListener('change', hideEmail);
+        hideEmail();
+        $('.datepicker').datetimepicker({
+            timepicker: false,
+            format: 'Y-m-d'
+        });
```

Comment:
Replace this with (this brings it into openemr's translation/internationalization engine):
``````
    $('.datepicker').datetimepicker({
        <?php $datetimepicker_timepicker = false; ?>
        <?php $datetimepicker_showseconds = false; ?>
        <?php $datetimepicker_formatInput = false; ?>
        <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
        <?php // can add any additional javascript settings to datetimepicker here; need to prepend first setting with a comma ?>
    });